app/utils/export.py

```diff
@@ -0,0 +1,103 @@
+from datetime import datetime
```

Comment:
What is the difference between files under `internal/` and `utils/`  folders?

Reply:
+1. Maybe should be under app/routers/export.py?